update the ide

mirnawong1 opened the pull request:
Added the new feature of reclone repo in the ide with added video

## Description & motivation
<!---
Describe your changes, and why you're making them. Is this linked to an open
issue, a pull request on dbt core, etc?
-->

## To-do before merge
<!---
(Optional -- remove this section if not needed)
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Change the base branch
- [ ] Ensure PR #56 is merged
-->

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

mirnawong1 committed:
update the ide 

Added the new feature of reclone repo in the ide with added video

runleonarun committed:
Update the-dbt-ide.md

runleonarun commented:
@mirnawong1 I made some [changes to this PR](https://github.com/dbt-labs/docs.getdbt.com/pull/1117/commits/1130c8d0ec2232e43cd0b385609701fecc0f9f1b), but I also can't see the video because it's too small. It also needs to be uploaded into `/img/docs/dbt-cloud/` Let me know if you want help with this!

mirnawong1 committed:
Update the-dbt-ide.md

Hi @runleonarun thank you very much and apologies about that, i've made that change here - is this the right format?

mirnawong1 commented:
duplicate and ide beta covers this
